src/python/tools/db2graph/marius_db2graph.py

```diff
@@ -0,0 +1,709 @@
+            exit(1)
+        
+        check_var = qry_split[3].lower()
+        if (check_var != "from"):
+            logging.error("Error: Incorrect edge entity node - entity node formatting, " +
+                "extra elements after table2_name.col2_name")
+            exit(1)
+        
+        new_query_list.append(edge_entity_entity_queries_list[q])
+    
+    return new_query_list
+
+def validation_check_edge_entity_feature_val_queries(edge_entity_feature_val_queries_list):
+    """
+    Ensures that the edge_entity_feature_val_queries_list are correctly formatted.
+
+    :param edge_entity_feature_val_queries_list: List of all the queries defining edges from entity node to feature values
+    :return new_query_list: These are updated queries with necessary changes if any
+    """
+    # Format: SELECT table1_name.col1_name, ____ FROM ____ WHERE ____ (and so on);
+    new_query_list = list()
+    for q in range(len(edge_entity_feature_val_queries_list)):
+        qry_split = edge_entity_feature_val_queries_list[q].split(' ')
+        
+        check_var = qry_split[0].lower()
+        if (check_var != "select"):
+            logging.error("Error: Incorrect edge entity node - feature value formatting, " +
+                "not starting with SELECT")
+            exit(1)
+        
+        check_split = qry_split[1].split('.')
+        if (len(check_split) != 2):
+            logging.error("Error: Incorrect edge entity node - feature value formatting, " +
+                "table1_name.col1_name not correctly formatted")
+            exit(1)
+        if (check_split[1][-1] != ','):
+            logging.error("Error: Incorrect edge entity node - feature value formatting, " +
+                "missing ',' at the end of table1_name.col1_name")
+            exit(1)
+        
+        new_query_list.append(edge_entity_feature_val_queries_list[q])
+    
+    return new_query_list
+
+def clean_token(token):
+    """
+    Helper to clean a dataframe, can be used by applying this function to a dataframe
+
+    :param token: elements to clean
+    :return token: cleaned token
+    """
+    token = str(token)
+    token = token.strip().strip("\t.\'\" ")
+    return token.lower()
+
+def get_uuid(row):
+    """
+    Gets the UUID associated with entity_node on this row consisting of a three-tuple
+
+    :param row: a row in the dataframe
+    :return uuid: the UUID associated with entity_node on this row
+    """
+    return uuid.uuid5(uuid.NAMESPACE_DNS, row['entity_node'])
+
+def get_init_fetch_size():
+    """
+    In an initial pass, estimates the optimal maximum possible fetch_size for given query based on memory usage report of virtual_memory() 
+
+    :return limit_fetch_size: the optimal maximum possible fetch_size for database engine
+    """
+    mem_copy = psutil.virtual_memory()
+    mem_copy_used = mem_copy.used
+    limit_fetch_size = min(mem_copy.available / 2, 1000000000) # max fetch_size limit at one billion due to database constraint
```

Comment:
nit: `1000000000` can be a constant at the top of the file. Also you can use scientific notation: `1E9` which is a bit easier to read for big numbers

Reply:
Updated as constant in scientific notation